.circleci/config.yml

```diff
@@ -78,6 +78,13 @@ jobs:
           mkdir -p $GRPC_DEST
           git clone --branch v1.2.0 https://github.com/grpc/grpc-go $GRPC_DEST
 
+    - run:
+        name: Vendor redis v6.15.3
+        # This step vendors go-redis/redis v6.15.3 inside our redis contrib
+        # to allow running the tests against the correct version of
+        # the redis library.
+        command: |
+          git clone --branch v6.15.3 https://github.com/go-redis/redis contrib/go-redis/redis/vendor/github.com/go-redis/redis
```

Comment:
I think we have to rename the contribution package to `contrib/go-redis/redis.v6.15.3` just as we do for grpc, and make a copy of it inside `contrib/go-redis/redis` which supports the latest. The latter part you can skip if you think it's too much work and we can do it as part of another PR. But nevertheless we should do it immediately after.

Reply:
Right, as discussed on Slack: it seems there is no v7 release yet, so there is no point to follow this comment for now. Resolving.